.github/workflows/test.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile --network-timeout 1000000
 
+      - name: Lint
+        run: yarn lint
```

Comment:
Run `yarn lint` on PRs now - will only fail if ERROR rules fail, and we should only be using ERROR for functional eslint rules.  Stylistic eslint rules should be WARN and will get auto-fixed after merges